rosbridge_library/src/rosbridge_library/internal/publishers.py

```diff
@@ -320,8 +320,9 @@ def unregister(self, client_id, topic):
 
     def _unregister_impl(self, topic):
         if not self._publishers[topic].has_clients():
-            self._publishers[topic].unregister()
-            del self._publishers[topic]
+            #self._publishers[topic].unregister()
+            #del self._publishers[topic]
```

Comment:
Instead of commenting out code lines, I'd add a link to where this hack was taken from for future trackability.

Reply:
Done